### PR TITLE
chore: update RWA to make running locally easier

### PR DESCRIPTION
**Pull request by AtofStryker**
I noticed after running the RWA earlier this week that there were some typing, linting, and other issues that make the app somewhat difficult to run locally. This PR addresses the typing issues, dependency issues, and old scaffolded data.

I had to downgrade `passport` due to https://github.com/jaredhanson/passport/issues/904

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -116,6 +116,7 @@ Cypress.Commands.add("reactComponent", { prevSubject: "element" }, ($el) => {
 });
 
 Cypress.Commands.add("setTransactionAmountRange", (min, max) => {
+  // eslint-disable-next-line cypress/unsafe-to-chain-command
```
Comment (cacieprins) [inline cypress/support/commands.ts]: I think this works fine if we just stop chaining click onto scrollIntoView here. If this linter error isn't necessary, we should probably have a larger discussion about it.

**Review by cacieprins — CHANGES_REQUESTED** [review]

File: cypress/support/commands.ts (hunk 1 above)
Comment (AtofStryker) [reply cypress/support/commands.ts]: we could just refactor this to 
```
cy.getBySel("transaction-list-filter-amount-range-button")
    .scrollIntoView()
cy.getBySel("transaction-list-filter-amount-range-button").click({ force: true });
```
but it seems a bit ridiculous that we are scrolling to something then forcing the click?

**Review by cacieprins — APPROVED** [review]

File: cypress/support/commands.ts (hunk 1 above)
Comment (AtofStryker) [reply cypress/support/commands.ts]: addressed in 71c38c9

**Comment by MikeMcC399** [comment]
@AtofStryker 

- Unfortunately this PR breaks RWA on Windows (see issue #1529).

**Pull request merged**